app/views/facility_journals/_problem_transactions.html.haml

```diff
@@ -3,6 +3,9 @@
     %p.alert.alert-error= t("facility_journals.new.expired_payment")
 .row
   .span12
+    %p= link_to t("reports.account_transactions.export"), url_for(params.merge(format: :csv)), class: 'js--exportSearchResults pull-right', data: { form: '.search_form' }
```

Comment:
Still gonna flag on @date_range_field (instance variable in a partial), unless we want to break the search functionality.